packages/@n8n/nodes-langchain/nodes/vector_store/shared/createVectorStoreNode.ts

```diff
@@ -395,7 +396,14 @@ export const createVectorStoreNode = (args: VectorStoreNodeConstructorArgs) =>
 
 			if (mode === 'retrieve') {
 				const vectorStore = await args.getVectorStoreClient(this, filter, embeddings, itemIndex);
+
+				async function closeFunction() {
```

Comment:
Can you revert the changes to this file, and merge back the latest master?
I think the `closeFunction` removal in the other file is all that's needed now after #12346

Reply:
If I don’t include this change, PGVectorStore will also increase the connections to Postgres. I couldn’t find a cleaner way to apply this adjustment. I ran tests by removing closeFunction, but the connections still increased.”